x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/middleware.test.ts

```diff
@@ -73,7 +73,8 @@ const createStoreSetup = (trustedAppsService: TrustedAppsService) => {
   };
 };
 
-describe('middleware', () => {
+// Skipped during Jest 29 upgrade
+describe.skip('middleware', () => {
```

Comment:
I'm not entirely sure why only this suite is failing. I thought it may be due to changes in mocking and how Redux is handled. Buy a very similar [suite](https://github.com/elastic/kibana/blob/7.17/x-pack/plugins/security_solution/public/management/pages/event_filters/store/middleware.test.ts) is passing fine.

Reply:
I think we can open an issue to let the underlying team know about the skip and proceed with this PR